### PR TITLE
Initial

### PR DESCRIPTION
**Pull request by josex2r**

### CODE REVIEW

File: .eslintrc.js [hunk 1]
```diff
@@ -1,11 +1,16 @@
 module.exports = {
```
Comment (etarancon) [inline .eslintrc.js:3]: Add 'use strict';

**Review by etarancon — APPROVED** [review]

File: index.js [hunk 2]
```diff
@@ -1,5 +1,56 @@
...
+		if (checker.for('ember-qunit', 'npm').exists() || checker.for('ember-cli-qunit', 'npm').exists()) {
+			this._testGenerator = 'qunit';
+		} else if (checker.for('ember-mocha', 'npm').exists() || checker.for('ember-cli-mocha', 'npm').exists()) {
+			this._testGenerator = 'mocha';
+		}
+	},
+
+	included(app) {
+		this._super.included.apply(this, arguments);
+		this._options = app.options.remark || {};
```
Comment (beatrizdemiguelperez) [inline index.js:28]: I would get the options from 'ember-cli-remark' instead of 'remark' because Those are options for the addon no for the remark library, i mean, when you create remark you pass only quiet and testGenerator. Its similar to ember-cli-workbox, you habe "ember-cli-workbox" options and "workbox" options.

**Review by xe21500 — APPROVED** [review]

File: node-tests/test.js [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+		return emberTest().then((result) => {
+			expect(result.error).to.not.exist;
+			expect(result.stdout).to.match(/^ok 1 Chrome \d+.0 - RemarkLint \| CONTRIBUTING.md: should pass RemarkLint/m);
+			expect(result.stdout).to.match(/^ok 2 Chrome \d+.0 - RemarkLint \| LICENSE.md: should pass RemarkLint/m);
+			expect(result.stdout).to.match(/^ok 3 Chrome \d+.0 - RemarkLint \| README.md: should pass RemarkLint/m);
+			expect(result.stdout).to.not.match(/^not ok 8 Chrome \d+.0 - RemarkLint \| tests\/dummy\/app\/unused.md: should pass RemarkLint/m);
+		});
+	});
+
+	it('fails if a RemarkLint tests fails', () => {
```
Comment (beatrizdemiguelperez) [inline node-tests/test.js]: here you can use async await to wait for embertest promise

File: node-tests/test.js [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+			expect(result.stdout).to.match(/^not ok 8 Chrome \d+.0 - RemarkLint \| tests\/dummy\/app\/unused.md: should pass RemarkLint/m);
+
+		});
+	});
+});
+
+function emberTest() {
+	return new Promise((resolve) => {
+		exec('node_modules/.bin/ember test', { cwd: path.join(__dirname, '..'), env: process.env }, (error, stdout, stderr) => {
+			resolve({
```
Comment (beatrizdemiguelperez) [inline node-tests/test.js:45]: I would reject promise if exec fails

File: package.json [hunk 5]
```diff
@@ -45,14 +55,16 @@
     "ember-load-initializers": "^1.0.0",
     "ember-maybe-import-regenerator": "^0.1.6",
     "ember-resolver": "^4.0.0",
-    "ember-source": "~3.0.0",
+    "ember-source": "~2.12.0",
```
Comment (beatrizdemiguelperez) [inline package.json:58]: why did you change ember-source version?

**Review by beatrizdemiguelperez — CHANGES_REQUESTED** [review]

**Review by beatrizdemiguelperez — APPROVED** [review]
